src/main/java/com/app/screen/controller/MainController.java

```diff
@@ -355,9 +371,18 @@ public void initialize(URL location, ResourceBundle resources) {
     }
 
     private void initializeUpper() {
+
+        String displayA = ep.getByType(GraphType.UPPER).getDisplayYY().getA();
+
+        if(displayA.equals("|Y|") || displayA.equals("|Z|") || displayA.equals("|r|")){
```

Comment:
@MariaRiya toto nie je velmi najlepsie riesenie :/ abs by mohla byt ako bool hodnota v displayYY a potom spravit nejaku static method do DisplayA ktora vrati ci treba kontrolovat abs alebo nie

Reply:
vyriesime to zajtra po testovani